yarn.lock

```diff
@@ -10525,7 +10525,7 @@ aws-sdk-mock@^5.8.0:
     sinon "^14.0.1"
     traverse "^0.6.6"
 
-aws-sdk@^2.1231.0, aws-sdk@^2.1354.0:
+aws-sdk@^2.1231.0, aws-sdk@^2.1350.0, aws-sdk@^2.1354.0:
```

Comment:
where does `2.1350.0` come from?. If it's our code then we should bump it to `2.1354.0` in package.json.

Reply:
updated the PR to keep yarn.lock unchanged